cirq-core/cirq/circuits/circuit_operation.py

```diff
@@ -40,40 +40,18 @@ def default_repetition_ids(repetitions: int) -> Optional[List[str]]:
     return None
 
 
-def cartesian_product_of_string_lists(list1: Optional[List[str]], list2: Optional[List[str]]):
+def full_join_string_lists(list1: Optional[List[str]], list2: Optional[List[str]]):
```

Comment:
Should this be a private method (i.e. with leading "_")?

Reply:
The test uses it. Not sure what the python convention for this is. Is it okay for unit tests to depend on private methods?